client/main.lua

```diff
@@ -199,8 +199,19 @@ local function deletePeds()
     pedsSpawned = false
 end
 
--- Events
+local function getJobs()
+    QBCore.Functions.TriggerCallback('qb-cityhall:server:receiveJobs', function(result)
+        SendNUIMessage({
+            action = 'setJobs',
+            jobs = result
+        })
+    end)
+end
 
+-- Events
+RegisterNetEvent('cityhall:Client:AddCityJob', function(jobName,label)
```

Comment:
Incorrect event name, should be `qb-cityhall`

Reply:
Done